packages/calcite-components/src/components/tabs/tabs.tsx

```diff
@@ -45,6 +52,19 @@ export class Tabs {
   //
   //--------------------------------------------------------------------------
 
+  connectedCallback(): void {
+    this.mutationObserver?.observe(this.el, { childList: true, subtree: true });
```

Comment:
I'd like to avoid using `subtree` here since it means the observer will run on every child update within the components (i.e., all tab-related components + their respective content). Some alternatives would be:

1. update `tab-nav` to forward relevant properties to its `tab-title` children
2. update the mutation observer callback to bail if the mutation target is not a tab-related component
3. reuse the tabs observer on its tab-nav

Reply:
Going with: update the mutation observer callback to bail if the mutation target is not a tab-related component.